Resources/Public/CSS/qucosa.css

```diff
@@ -289,6 +299,11 @@
     left: 1px;
 }
 
+.tx-dpf .glyphicon-remove {
+    top: 1.5px !important;
```

Comment:
The "!important" declaration should not appear in production code, should it? It's fine for now, since we are going to reinvent the whole CSStuff with 4.x

Reply:
I had to set this rule explicitly or a more specific css rule would take precedence and my change wouldn't have any effect.

But you're right. I checked again and I can get rid of this `!important`.